lib/ansible/plugins/lookup/lmdb_kv.py

```diff
@@ -0,0 +1,81 @@
+# (c) 2017, Jan-Piet Mens <jpmens(at)gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+from ansible.errors import AnsibleError
+from ansible.plugins.lookup import LookupBase
+
+HAVE_LMDB = True
+try:
+    import lmdb
+except ImportError:
+    HAVE_LMDB = False
+
+
```

Comment:
We now require that lookup plugins have documentation to be included in Ansible

Reply:
a year later? honestly?

Reply:
sorry for the delay, but this one was dropped in my queue as it was closed/reopppened

Reply:
We now have docs team reviewing all the docs. 
@jpmens Do you have time to work on this?

Reply:
I do not, @gundalow, sorry.

Reply:
documentation added - thanks @jpmens